src/components/Notifications.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+    <v-bottom-sheet :model-value="hasAnyPushedNotificationsThatShouldBeVisible">
+        <v-card>
+            <v-card-item>
+                <v-card-title>Notifications</v-card-title>
+            </v-card-item>
+            <v-card-text>
+                <div class="h-auto d-flex flex-column align-stretch ga-2">
+                    <v-alert
+                        v-for="notification in pushedNotifications"
+                        :closable="notification.closable"
+                        :density="notification.density"
+                        :text="notification.text"
+                        :title="notification.title"
+                        :type="notification.type"
+                        v-model="notification.shouldBeVisible"
+                    ></v-alert>
+                </div>
+            </v-card-text>
+            <v-card-actions>
+                <v-spacer></v-spacer>
+                <v-btn
+                    prepend-icon="mdi-close"
+                    @click="closeAllPushedNotifications"
+                    >Close All</v-btn
+                >
+            </v-card-actions>
+        </v-card>
+    </v-bottom-sheet>
+</template>
+
+<script lang="ts" setup>
+import { Notification, useAppStore } from '@/store/app'
+import { storeToRefs } from 'pinia'
+import { computed, nextTick, ref, watch } from 'vue'
+
+/**
+ * Interface representing an alert to be displayed.
+ *
+ * @property {boolean} shouldBeVisible - Indicates whether the alert should be visible.
```

Comment:
you don't need to document the types in jsdoc syntax when you ae already using types in the interface definition.
does not do any harm, just for the future

Reply:
Good to know! Thank you!